### PR TITLE
transition was added

### PR DESCRIPTION
**Pull request by Yuliia-Hrinchenko1**
- [DEMO LINK](https://Yuliia-Hrinchenko1.github.io/layout_catalog_hovers/)

### CODE REVIEW

**Review by Stepan-TS — APPROVED** [review]
well done!)

File: src/styles/utils/_variables.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+$transiotion-duration: 300ms;
```
Comment (sergii-nosachenko) [inline src/styles/utils/_variables.scss:1]:
```suggestion
$transition-duration: 300ms;
```

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -17,9 +18,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transiotion-duration;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/catalog.scss:21]: Add the `transition-property` prop with the correct values.

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $transiotion-duration;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/header.scss:41]: Here too

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss [hunk 4]
```diff
@@ -18,7 +18,8 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-    transition-duration: $transiotion-duration;
+    transition-duration: transform $transition-duration,
```
Comment (sergii-nosachenko) [inline src/styles/blocks/catalog.scss:21]:
```suggestion
    transition: transform $transition-duration,
```

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -38,7 +38,7 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
-    transition-duration: $transiotion-duration;
+    transition-duration: color $transition-duration;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/header.scss:41]: Wrong prop. Use [transition shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/transition).

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

File: src/styles/blocks/header.scss (hunk 5 above)
Comment (Yuliia-Hrinchenko1) [reply src/styles/blocks/header.scss]: I hope my code is okay now))

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**